pkg/apis/management.cattle.io/v3/schema_types.go

```diff
@@ -5,75 +5,322 @@ import (
+	//
+	// No action handlers can be defined for dynamic schemas, so the
+	// behavior of these actions cannot be defined.
+	// +optional
+	CollectionActions map[string]Action `json:"collectionActions,omitempty"`
+
+	// CollectionFilters are filters for listing collections of resources of
+	// this schema. Each map key is a field that can be used as a filter.
+	//
+	// No Norman stores are defined for dynamic schemas, so no collections
+	// can be retrieved to be filtered.
+	// +optional
+	CollectionFilters map[string]Filter `json:"collectionFilters,omitempty"`
+
+	// IncludeableLinks has no effect.
+	// +optional
+	IncludeableLinks []string `json:"includeableLinks,omitempty"`
+
+	// DynamicSchemaVersion is used to determine whether the Norman schema
+	// should be updated when this object is updated. The schema is only
+	// updated if the previous version differs. Rancher overwrites this
+	// field.
+	// +optional
+	DynamicSchemaVersion string `json:"dynamicSchemaVersion,omitempty"`
 }
 
 type DynamicSchemaStatus struct {
+	// Fake has no effect.
+	// +optional
 	Fake string `json:"fake,omitempty"`
 }
 
 type Field struct {
-	Type         string   `json:"type,omitempty"`
-	Default      Values   `json:"default,omitempty"`
-	Unique       bool     `json:"unique,omitempty"`
-	Nullable     bool     `json:"nullable,omitempty"`
-	Create       bool     `json:"create,omitempty"`
-	Required     bool     `json:"required,omitempty"`
-	Update       bool     `json:"update,omitempty"`
-	MinLength    int64    `json:"minLength,omitempty"`
-	MaxLength    int64    `json:"maxLength,omitempty"`
-	Min          int64    `json:"min,omitempty"`
-	Max          int64    `json:"max,omitempty"`
-	Options      []string `json:"options,omitempty"`
-	ValidChars   string   `json:"validChars,omitempty"`
-	InvalidChars string   `json:"invalidChars,omitempty"`
-	Description  string   `json:"description,omitempty"`
-	DynamicField bool     `json:"dynamicField,omitempty"`
+	// Type is the type of the field. Possible types are "string",
+	// "password", "boolean", "int", "array[string]" or another schema name.
+	// +optional
+	Type string `json:"type,omitempty"`
+
+	// Default is the default value for the field.
+	// +optional
+	Default Values `json:"default,omitempty"`
+
+	// Unique has no effect.
+	// +optional
+	Unique bool `json:"unique,omitempty"`
+
+	// Nullable indicates that a field in a CRD derived from this schema
+	// will be nullable.
+	//
+	// It also indicates whether the field can take a null value in a valid
+	// object of this schema in the Norman API.
+	//
+	// No Norman stores are defined for DynamicSchemas, so no resources of
+	// this schema can be created in the Norman API.
+	// +optional
+	Nullable bool `json:"nullable,omitempty"`
+
+	// Create indicates that this field should not be set when creating a
+	// new object of this schema through the Norman API. A value for this
+	// field will be filtered out if present in the create request.
```

Comment:
Please let me know if I understand it correctly :) 

```suggestion
	// Create indicates that this field should be excluded when creating a
	// new object of this schema through the Norman API. If a value is provided for this field
	// in a create request, it will be filtered out and ignored.
```

Reply:
That's right, thanks. I've added a similar sentence to update as well.